docs/controls/OrbitView.md

```diff
@@ -2,7 +2,7 @@
 
 The `OrbitView` control provides a new control, inherited from the `ItemsControl`. All items are arranged in circle around a center element
 
-![OrbitView Overview](../resources/images/OrbitView.gif "OrbitView")  
+![OrbitView Overview](../resources/images/Controls-OrbitView.png "OrbitView")  
```

Comment:
I found the gif is named SpaceVIew.gif and not OrbitView.gif. Can we rename and then use the gif?